Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -1,96 +1,67 @@
-Enable Hubble
-==============
+Enable Hubble for Cluster-Wide Visibility
+=========================================
 
-Hubble is a fully distributed networking and security observability platform
-for cloud native workloads. It is built on top of Cilium and eBPF to enable
-deep visibility into the communication and behavior of services as well as the
-networking infrastructure in a completely transparent manner.
+Hubble is the component for observability in Cilium. Cluster-wide visibility is
```

Comment:
while following the instructions, this paragraph felt a bit out of place. it might make it a bit smoother if we move this to the bottom and simply say:
```
.. note:

   See the :ref:`concepts_observability` section for more information about observability in Cilium.
```